### DISCUSSION TITLE
Allow hyperfruit not forced to dash in the same timestamp

### DISCUSSION THREAD

**Discussion by Primakien** [ppy/osu#26179] [Q&A] [answered]
https://github.com/ppy/osu/assets/100775884/9f5fe19c-f37c-45d2-80b1-7233ed2c54a6

Stable version doesn't dash the catcher but in lazer it's dash for some reason even 2 fruits placing far away in the same timestamp, making it quite annoying to catch in the middle of 2 fruits.

**Comment by bdach** [comment]
please link beatmap

**Reply by Primakien** [reply]
https://osu.ppy.sh/beatmapsets/1365009#fruits/2875589

**Reply by peppy** [reply, marked as answer]
Tracking at https://github.com/ppy/osu/issues/26537.
